wiki/Disambiguation/Snapping/id.md

```diff
@@ -0,0 +1,13 @@
+# Snapping (disambiguasi)
+
+**Snapping** dapat memiliki beberapa arti:
+
+## Beatmapping
+
+- [Beat snapping](/wiki/Beatmapping/Snapping), yang merupakan posisi sebuah hit object pada sebuah ketukan musik yang telah dibagi secara otomatis oleh [beat snap divisor](/wiki/Client/Beatmap_editor/Beat_Snap_Divisor)
```

Comment:
```suggestion
- [Beat snapping](/wiki/Beatmapping/Snapping), posisi dari sebuah *hit object* pada satu ketukan musik, yang dibagi secara otomatis menggunakan [beat snap divisor](/wiki/Client/Beatmap_editor/Beat_Snap_Divisor)
```

Reply:
- there is no `is` usage
- italic still a thing in ID?
- `sebuah` should be used separately to one phrase
- no past tense here
- `menggunakan` should be relateable since used from hand humans.